templates/hooks/key/sdk_create_post_set_output.go.tpl

```diff
@@ -2,4 +2,8 @@
     if err != nil {
         return &resource{ko}, err
     }
-    ko.Spec.Policy = policy
\ No newline at end of file
+    ko.Spec.Policy = policy
+    err = rm.updateKeyRotation(ctx, &resource{ko})
+    if err != nil {
+        return &resource{ko}, err
+    }
```

Comment:
We also need a post "read many hook" that queries the key rotation status and sets it to `ko.Spec.EnableKeyRotation`

Reply:
Can you point me to the reference for this? I am still new to this template thing.

Reply:
Sorted now @A-Hilaly